src/main.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+
+ReactDOM.createRoot(document.getElementById('root')).render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>
```

Comment:
- [ ] make sure you wrap the parent app with the Router component `<BrowserRouter>`, It is a type of router that uses the HTML5 history API to keep the URL in sync with the view. With this router, you are sure of having a clean URL in the browser address bar 👍 😄

Reply:
I think this is done only when you are to add the different routes and in my case i did not add any